src/actions/receive-profile.js

```diff
@@ -384,6 +392,10 @@ async function getSymbolStore(
       );
     },
     requestSymbolTableFromAddon: async lib => {
+      if (!geckoProfiler) {
+        throw new Error("There's no connection to the gecko profiler add-on.");
+      }
+
```

Comment:
This is what makes it possible to do a symbolication process when we don't have a geckoProfiler addon.